### PR TITLE
Update Xed Main Page & Build Documentation

### PR DESCRIPTION
**Pull request by kkhalail**
update doxygen version to 1.8.17

### CODE REVIEW

**Comment by kkhalail** [comment]
updated main page
image

**Comment by kkhalail** [comment]
compared build doc: removal of obsolete build knobs, addition of enc2 explanation, python version updates in build cmds

File: build-manual/arrowdown.png
Comment (marjevan) [inline build-manual/arrowdown.png:1]: Do we need it?

File: build-manual/index.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
 <meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
 <meta http-equiv="X-UA-Compatible" content="IE=9"/>
-<meta name="generator" content="Doxygen 1.8.13"/>
+<meta name="generator" content="Doxygen 1.8.17"/>
 <meta name="viewport" content="width=device-width, initial-scale=1"/>
-<title>XED-BUILD: XED Build Guide</title>
+<title>XED-BUILD: &lt;em&gt;EXTERNAL&lt;/em&gt; XED Build Guide</title>
```
Comment (marjevan) [inline build-manual/index.html]: I don't see any point in the "EXTERNAL" addition

File: build-manual/index.html [hunk 2]
```diff
@@ -26,23 +26,23 @@
...
 });
-</script>
+/* @license-end */</script>
 <div id="main-nav"></div>
 </div><!-- top -->
-<div class="header">
+<div class="PageDoc"><div class="header">
   <div class="headertitle">
-<div class="title">XED Build Guide </div>  </div>
+<div class="title"><em>EXTERNAL</em> XED Build Guide </div>  </div>
```
Comment (marjevan) [inline build-manual/index.html]: same

File: build-manual/index.html [hunk 3]
```diff
@@ -26,23 +26,23 @@
...
 </div><!--header-->
 <div class="contents">
-<div class="textblock"><p>by Mark Charney</p>
-<p>2019-02-04</p>
+<div class="textblock"><p>2019-05-28</p>
```
Comment (marjevan) [inline build-manual/index.html:45]: Update

File: build-manual/index.html (hunk 1 above)
Comment (kkhalail) [reply build-manual/index.html]: Mark added this 4 years ago - @if  (EXTERNAL)
@mainpage *EXTERNAL* XED Build Guide
@else
@mainpage *INTERNAL* XED Build Guide
@endif

File: build-manual/index.html (hunk 3 above)
Comment (kkhalail) [reply build-manual/index.html:45]: yes this is already in my bucket list for next changes, as they require XED PR

File: build-manual/arrowdown.png
Comment (kkhalail) [reply build-manual/arrowdown.png:1]: there is no mention of it in the documentation, unlike the other images (that weren't deleted) that means it's not needed for the GUI

**Review by marjevan — APPROVED** [review]

**Pull request merged**